docs/book/src/user/quick-start.md

```diff
@@ -376,28 +376,6 @@ ssh-keygen -t rsa -b 2048 -f "${SSH_KEY_FILE}" -N '' 1>/dev/null
 echo "Machine SSH key generated in ${SSH_KEY_FILE}"
 export AZURE_SSH_PUBLIC_KEY=$(cat "${SSH_KEY_FILE}.pub" | base64 | tr -d '\r\n')
 
-# To populate secret in azure.json file.
-export AZURE_JSON_B64=$(echo "{
-    \"cloud\": \"${AZURE_ENVIRONMENT}\",
-    \"tenantId\": \"${AZURE_TENANT_ID}\",
-    \"subscriptionId\": \"${AZURE_SUBSCRIPTION_ID}\",
-    \"aadClientId\": \"${AZURE_CLIENT_ID}\",
-    \"aadClientSecret\": \"${AZURE_CLIENT_SECRET}\",
-    \"resourceGroup\": \"${CLUSTER_NAME}\",
-    \"securityGroupName\": \"${CLUSTER_NAME}-node-nsg\",
-    \"location\": \"${AZURE_LOCATION}\",
-    \"vmType\": \"vmss\",
-    \"vnetName\": \"${AZURE_VNET_NAME}\",
-    \"vnetResourceGroup\": \"${CLUSTER_NAME}\",
-    \"subnetName\": \"${CLUSTER_NAME}-node-subnet\",
-    \"routeTableName\": \"${CLUSTER_NAME}-node-routetable\",
-    \"loadBalancerSku\": \"standard\",
-    \"maximumLoadBalancerRuleCount\": 250,
-    \"useManagedIdentityExtension\": false,
-    \"useInstanceMetadata\": true
-}" | base64 | tr -d '\r\n')
-```
-
```

Comment:
should we mention that you can create your own secret if you have specific values, or is that too complicated for quick-start?

Reply:
I think that's out of scope for quick start since it uses the default flavor. I only expect very advanced users to want to write their own azure.json config as 99% of use cases should be exposed in the AzureCluster spec.

Reply:
sgtm